### DISCUSSION TITLE
Tailwind CSS on languages

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#3768] [Ideas]
It would be cool to have a [Tailwind CSS](https://tailwindcss.com) classes language in Monkeytype. I know, there is now `code css` but typing test with Tailwind CSS classes are cool!

**Comment by Miodec** [comment]
If you prepare a list of the classes I can add it.

**Reply by ghost** [reply]
Is it ok if I can only have around 200 tailwind classes? I don't know where can I find all tailwind classes so I just generated one manually 😅. Also, can I just send it here? Thanks.

**Reply by Miodec** [reply]
What do you mean generated one manually?

**Reply by ghost** [reply]
Like adding all mostly used tailwind classes on a text file one by one. That's what I meant.
